schemas/v1/5_miner_sector_snaps.go

```diff
@@ -0,0 +1,59 @@
+package v1
+
+func init() {
+	patches.Register(
+		5,
+		`
+	-- add new sector event type for snapped sectors
+	ALTER TYPE {{ .SchemaName | default "public"}}.miner_sector_event_type ADD VALUE 'SECTOR_SNAPPED' AFTER 'SECTOR_TERMINATED';
+
+	-- ----------------------------------------------------------------
+	-- Name: miner_sector_infos_v7plus
+	-- Model: miner.MinerSectorInfoV7Plus
+	-- Growth: About 180 per epoch
```

Comment:
This is not accurate for mainnet. Here's a sample of the epochs with the largest miner sector info counts:

<details>
<summary>Sector infos counts by height (sorted)</summary>

```csv
epoch   count
1308103	62862
1308104	55310
1351277	52732
1351278	48938
957453	44865
957442	44711
1310880	43818
1310869	41598
1308087	39744
1351276	38868
1308085	37501
1308099	36948
1310890	36937
1432490	36692
1308127	35628
1308132	34858
1308106	34773
1310892	34767
1308168	34713
957255	34077
1308113	32569
1310888	32549
1308167	32439
1308147	32433
1308112	32365
1310868	32331
1052506	32193
1135670	31827
1307986	30727
1308101	30246
1308098	30230
1308161	30168
1310894	30118
1308146	30093
1308169	30080
1308141	30060
1308128	30057
1308109	29929
1357687	28750
1308008	28629
1308001	28568
1307995	28451
1308014	28365
806834	28344
1308088	28266
957467	27998
```

</details>

Reply:
What do you think a good value here is then?